Packs/Rapid7_InsightIDR/Integrations/Rapid7_InsightIDR/insightidr.py

```diff
@@ -0,0 +1,836 @@
+"""Rapid7 InsightIDR Investigation for Cortex XSOAR."""
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import json
+import urllib3
+from datetime import datetime, timedelta
+from typing import Dict, Tuple
+from requests import Response
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
+INVESTIGATIONS_FIELDS = ['title', 'id', 'status', 'created_time', 'source', 'assignee', 'alerts']
+THREATS_FIELDS = ['name', 'note', 'indicator_count', 'published']
+LOGS_FIELDS = ['name', 'id']
+
+EVENTS_FIELDS = ['log_id', 'message', 'timestamp']
```

Comment:
as they are used only once, please put each in the relevant func as a lower case var

Reply:
All of them are used at least in two different functions, should I make the change anyway?

Reply:
my bad, dismissed.